src/NextjsBuild.ts

```diff
@@ -103,9 +103,28 @@ export class NextjsBuild extends Construct {
     super(scope, id);
     this.props = props;
     this.validatePaths();
-    // when `cdk deploy "NonNextjsStack" --exclusively` is run, don't run build
-    if (Stack.of(this).bundlingRequired && !this.props.skipBuild) {
-      this.build();
+
+    const bundlingRequired = Stack.of(this).bundlingRequired;
+    const skipBuild = this.props.skipBuild;
+
+    /**
```

Comment:
Instead of an inline comment, do you mind adding this helpful table to docs/code-deployment-flow.md in a new section maybe titled "Conditional Build Logic"? Then maybe adding a brief explainer to accompany the table in the docs? Then you can just add a link to those docs in an inline comment. Future devs (including you, haha) will thank you :)

Reply:
sure